torch/matmul-relu-bias/main.py

```diff
@@ -0,0 +1,15 @@
+import torch
+from stardew.torch import torch_compiler
+
+
+# These should ideally also be converted to integer datatypes,
+# but torch-mlir does not support non floating-point types for
+# the RELU activation function
+
+x = torch.randint(-10, 10, (256, 128), dtype=torch.float32)
+w = torch.randint(-10, 10, (128,  10), dtype=torch.float32)
+b = torch.randint(-10, 10, ( 10,    ), dtype=torch.float32)
+
+@torch_compiler(example=[x, w, b])
+def foo(x : torch.Tensor, w : torch.Tensor, b: torch.Tensor):
+    return torch.relu(x @ w + b)
```

Comment:
Let's just run this as a test and an example at the same time?
That way we reduce code duplication :wink: